build.sh

```diff
@@ -18,16 +18,15 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean librmm rmm -v -g -n -s --ptds -h"
-HELP="$0 [clean] [librmm] [rmm] [-v] [-g] [-n] [-s] [--ptds] [-h]
+VALIDARGS="clean librmm rmm -v -g -n -s -h"
```

Comment:
I don't think you want to remove the option to build --ptds in this file, do you? You just want to remove the use of it in CI (the other file change you made). Somebody may want to use this to build the benchmarks with PTDS, for example?

Reply:
As far as I can tell this script doesn't allow you to build the benchmarks. The option `BUILD_BENCHMARKS` is set to `OFF` in the cmake file, and there is currently no way to turn it on through this script.

Reply:
OK. But if we fix that, all of these changes will have to be put back?

Reply:
I suppose so. But as it stands now, passing in `--ptds` has no effect, so it's kinda misleading.

Reply:
We should add the ability to build benchmarks to this script, could you leave the option in even if it has no effect for now?

Reply:
Done.